packages/node-sdk/paima-db/src/sql/events.sql

```diff
@@ -0,0 +1,51 @@
+/* @name getEvents */
+SELECT * FROM event WHERE
+  COALESCE(data->>:field0 = :value0, 1=1) AND
+  COALESCE(data->>:field1 = :value1, 1=1) AND
+  COALESCE(data->>:field2 = :value2, 1=1) AND
+  COALESCE(data->>:field3 = :value3, 1=1) AND
+  COALESCE(data->>:field4 = :value4, 1=1) AND
+  COALESCE(data->>:field5 = :value5, 1=1) AND
+  COALESCE(data->>:field6 = :value6, 1=1) AND
+  COALESCE(data->>:field7 = :value7, 1=1) AND
+  COALESCE(data->>:field8 = :value8, 1=1) AND
+  COALESCE(data->>:field9 = :value9, 1=1) AND
```

Comment:
So we only allow up to 10 fields in an event log max?

Reply:
Not really a limitation of how many there can be, this is just how many (of the indexed ones) can you filter in a rest request.

We can lift this limitation though, it's just that I couldn't think of a way of making it work for an arbitrary number of fields with pgtyped. There are array expansions and object expansions, but I couldn't find a syntax that would let me write this kind of condition. 

My best idea is to build the query dynamically (with parameters) and use `pg.Client` query directly. And I can keep this query and cast the return value to that, since it should be the same type.